lte/gateway/python/magma/pipelined/utils.py

```diff
@@ -86,8 +85,3 @@ def get_ovs_bridge(cls, datapath):
             logging.warning('Cannot initiate OVSDB connection: %s', e)
             return None
         return cls.ovs
-
-
-def get_virtual_iface_mac(iface):
```

Comment:
What was wrong with this one?

Reply:
There is nothing wrong with it, we are just trying to unify the way mac addresses are inferred in magma. Previously, we used `get_virtual_iface_mac` which depends on `netifaces` to get the mac from a network interface, and various `scapy` utilities gets the mac address from an IP. There was `get_mac_address` in `bridge_utils.py` which used some bash commands to get the mac from the network interface name.  Also, there was yet another utility in `ifaces.py`, which also relies on the same `netifaces` logic as `get_virtual_iface_mac`. As I am writing, I also discovered two other local `get_virtual_iface_mac` functions in `check_quota.py` and `redirect.py` which are identical to this one. It's not really clear why they should exist. We want to simplify this by using `getmac` everywhere.